flink-autoscaler-plugin-jdbc/src/test/java/org/apache/flink/autoscaler/jdbc/event/AbstractJdbcAutoscalerEventHandlerITCase.java

```diff
@@ -430,6 +447,75 @@ void testScalingEventWithParallelismChange() throws Exception {
                         });
     }
 
+    @MethodSource("getExpiredEventHandlersCaseMatrix")
+    @ParameterizedTest
+    void testCleanExpiredEvents(int expiredRecordsNum, Duration eventHandlerTtl) throws Exception {
+
+        final Clock clock = Clock.systemDefaultZone();
+        initExpiredEventHandlerRecords(eventHandlerTtl, expiredRecordsNum, clock);
+        final Timestamp createTimeOfUnExpiredRecord = Timestamp.from(clock.instant());
+        jdbcEventInteractor.createEvent(
+                "jobKey1",
+                "reason",
+                AutoScalerEventHandler.Type.Normal,
+                "message",
+                "eventKey",
+                createTimeOfUnExpiredRecord);
+
+        eventHandler = new JdbcAutoScalerEventHandler<>(jdbcEventInteractor, eventHandlerTtl);
+        eventHandler.cleanExpiredEvents();
+
+        // Check
+        try (ResultSet countResultSet =
+                        quickQuery("select count(1) from t_flink_autoscaler_event_handler");
+                ResultSet createTimeResultSet =
+                        quickQuery("select create_time from t_flink_autoscaler_event_handler")) {
+            assertThat(countResultSet).isNotNull();
+            assertThat(countResultSet.next()).isTrue();
+            assertThat(countResultSet.getInt(1)).isOne();
```

Comment:
Could we extract the unexpired record number into one parameter? It could cover more cases.